### PR TITLE
Adding Release notes for 3.1.2

### PR DESCRIPTION
**Pull request by poojaSprophecy**

### CODE REVIEW

**Review by slyons — APPROVED** [review]

File: docs/release_notes/july2023.md [hunk 1]
```diff
@@ -21,14 +79,14 @@ We are thrilled to announce the addition of support for conditional execution in
 
 With this update, users can now incorporate pass-through or removal conditions within a Gem, offering greater control and flexibility over data transformations. Conditional execution allows users to define specific criteria or rules that determine whether a particular transformation should be applied or skipped during Pipeline execution.
```
Comment (atbida) [inline docs/release_notes/july2023.md:80]: I suggest removing the first sentence "With this...transformations."

File: docs/release_notes/july2023.md [hunk 2]
```diff
@@ -21,14 +79,14 @@ We are thrilled to announce the addition of support for conditional execution in
...
-More details on this feature can be found [here](https:/docs.prophecy.io/low-code-spark/gems/conditional_execution).
+More details on this feature can be found [here](/docs/low-code-spark/gems/conditional-execution.md).
 
 #### Revamped Prophecy Fabrics UI
 
 We are excited to announce the revamped Prophecy Fabrics user interface (UI), designed to achieve feature parity with the Databricks cluster creation flow. This significant update ensures that users have a seamless experience and consistent options when creating clusters within Prophecy.
-The new UI introduces an enhanced Job sizes interface that captures all the options and flexibility provided by Databricks for cluster creation. These Job sizes can be leveraged to create clusters when execution Pipeline through Prophecy or scheduling Jobs through Prophecy.
+The new UI introduces an enhanced Job sizes interface that captures all the options and flexibility provided by Databricks for cluster creation. These Job sizes can be leveraged to create clusters when executing Pipeline through Prophecy or scheduling Jobs through Prophecy.
```
Comment (atbida) [inline docs/release_notes/july2023.md:87]: nitpick: "executing Pipelines"

File: docs/release_notes/july2023.md [hunk 3]
```diff
@@ -46,11 +104,11 @@ Previously, users could only connect a cluster to a single Pipeline, limiting th
 With this enhancement, users can now attach clusters to different Pipelines in different tabs, enabling concurrent connections without interference.
```
Comment (atbida) [inline docs/release_notes/july2023.md:104]: Do we now support having Prophecy open in multiple browser tabs?

File: docs/release_notes/july2023.md [hunk 4]
```diff
@@ -46,11 +104,11 @@ Previously, users could only connect a cluster to a single Pipeline, limiting th
...
 This improvement offers increased flexibility and productivity, allowing users to seamlessly switch between Pipeline configurations, perform parallel analyses, and efficiently manage data workflows.
 
-#### Alert and warning diagnostics for User passing plain text Username password in Gems
+#### Alert and warning diagnostics for Users passing plain-text Username password in Gems
 
 Using hardcoded usernames and passwords is not recommended when working with Gems.
 This approach can result in unsafe code, as the credentials may be exposed in your configured Git repository for the Project.
-We have added In place Warnings and Diagnostics if Users are using plain text Passwords in Gems. Read about the recommended usage [here](https:/docs.prophecy.io/low-code-spark/best-practices/use-dbx-secrets)
+We have added In place Warnings and Diagnostics if Users are using plain text Passwords in Gems. Read about the recommended usage [here](/docs/low-code-spark/best-practices/use-dbx-secret.md)
```
Comment (atbida) [inline docs/release_notes/july2023.md:111]: I suggest removing "In place"

**Review by atbida — APPROVED** [review]
Approving based on readability, with minor comments.

**Comment by atbida** [comment]
@poojaSprophecy do we now support passthrough authentication??!?!?
image

File: docs/release_notes/july2023.md (hunk 3 above)
Comment (poojaSprophecy) [reply docs/release_notes/july2023.md:104]: Yes we do

**Pull request merged**